vmdb/app/models/ems_openstack_infra.rb

```diff
@@ -16,7 +16,7 @@ def supports_port?
   end
 
   def supported_auth_types
-    %w(default amqp)
+    %w(default amqp keypair)
```

Comment:
@Fryguy right now I think for the host ssh, there is default and remote, so it doesn't really says for what it will be used. Only comment says the purpose. So I continued the generic way. :-)

I am fine with a different name.